python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -498,25 +509,59 @@ def do_evaluate(
                 colnames[0],
             )
         elif typ == "parquet":
-            filters = None
-            if predicate is not None and row_index is None:
-                # Can't apply filters during read if we have a row index.
-                filters = to_parquet_filter(predicate.value)
-            tbl_w_meta = plc.io.parquet.read_parquet(
-                plc.io.SourceInfo(paths),
-                columns=with_columns,
-                filters=filters,
-                nrows=n_rows,
-                skip_rows=skip_rows,
-            )
-            df = DataFrame.from_table(
-                tbl_w_meta.tbl,
-                # TODO: consider nested column names?
-                tbl_w_meta.column_names(include_children=False),
-            )
-            if filters is not None:
-                # Mask must have been applied.
-                return df
+            parquet_options = config.config.get("parquet_options", {})
+            if parquet_options.get("chunked", True):
+                reader = plc.io.parquet.ChunkedParquetReader(
+                    plc.io.SourceInfo(paths),
+                    columns=with_columns,
+                    nrows=n_rows,
+                    skip_rows=skip_rows,
```

Comment:
Concretely, why does `skip_rows != 0` not work? If there is a simple repro, it is a bug.

Reply:
```python
import pylibcudf as plc
import pyarrow as pa
import pyarrow.parquet as pq

path = "./test.parquet"
pq.write_table(pa.Table.from_pydict({'a':[1,2,3]}), path)

reader = plc.io.parquet.ChunkedParquetReader(
        plc.io.SourceInfo([path]),
        columns=['a'],
        nrows=-1,
        skip_rows=2,
        chunk_read_limit=0,
        pass_read_limit=17179869184 # 16 GiB
)

```

```
RuntimeError: CUDF failure at: /xyz/cudf/cpp/src/io/parquet/reader.cpp:44: skip_rows > 0 is not currently supported in the Chunked Parquet reader.
```

Reply:
@wence- I believe this issue is coming up through the tests that slice a scanned parquet file. The default `n_rows` value of `None` should lead to a `skip_rows` value of zero in most cases, so I think we should be good there. 

I think our impl is pushing the slice down into a skip_rows operation. If we could just not permit that unless the chunked reader is disabled, we might be ok, what do you think?

Reply:
See #16186

Reply:
OK, added a workaround.